test/integration/connect/envoy/case-wanfed-gw/secondary/server.hcl

```diff
@@ -7,6 +7,7 @@ primary_gateways = [
   "consul-primary:4431",
 ]
 primary_gateways_interval = "5s"
+retry_interval_wan        = "5s"
```

Comment:
This defaults to `30s` which is a great way to have the `TestEnvoy/case-wanfed-gw` test accidentally take long enough that the bats test framework assumes the test has failed.